test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -2718,6 +2718,105 @@ public void PackCommand_ContentInnerTargetExtension_AddsTfmSpecificContent(strin
+
+                    if (tfmProperty == "TargetFrameworks")
+                    {
+                        Assert.Contains(@"runtimes/win/lib/net46/abc.pdb", files);
+                    }
+                }
+            }
+        }
+
+        [PlatformFact(Platform.Windows)]
+        public void PackCommand_ContentInnerTargetExtension_SymbolFilesWithoutDll()
+        {
+            using (var testDirectory = msbuildFixture.CreateTestDirectory())
+            {
+                var projectName = "ClassLibrary1";
+                var workingDirectory = Path.Combine(testDirectory, projectName);
+                var projectFile = Path.Combine(workingDirectory, $"{projectName}.csproj");
+                Directory.CreateDirectory(workingDirectory);
+                File.WriteAllText(Path.Combine(workingDirectory, "abc.pdb"), "hello world");
+
+                msbuildFixture.CreateDotnetNewProject(testDirectory.Path, projectName, " classlib");
+
+                using (var stream = new FileStream(projectFile, FileMode.Open, FileAccess.ReadWrite))
+                {
+                    var xml = XDocument.Load(stream);
+                    var target = @"<Target Name=""CustomContentTarget"">
+    <ItemGroup>
+      <_TargetPathsToSymbolsWithTfm Include=""abc.pdb"">
+        <TargetPath>/runtimes/win/lib/$(TargetFramework)/abc.pdb</TargetPath>
+        <TargetFramework>$(TargetFramework)</TargetFramework>
+      </_TargetPathsToSymbolsWithTfm>
+    </ItemGroup>
+  </Target>";
+                    ProjectFileUtils.SetTargetFrameworkForProject(xml, "TargetFrameworks", "netstandard1.4;net46");
+                    ProjectFileUtils.AddProperty(xml, "TargetsForTfmSpecificDebugSymbolsInPackage", "CustomContentTarget");
+                    ProjectFileUtils.AddProperty(xml, "IncludeBuildOutput", "false", $"'$(TargetFramework)'=='netstandard1.4'");
+                    ProjectFileUtils.AddProperty(xml, "IncludeSymbols", "true");
+                    ProjectFileUtils.AddCustomXmlToProjectRoot(xml, target);
+                    ProjectFileUtils.WriteXmlToFile(xml, stream);
+                }
+
+                msbuildFixture.RestoreProject(workingDirectory, projectName, string.Empty);
+                msbuildFixture.PackProject(workingDirectory, projectName, $"/p:PackageOutputPath={workingDirectory}");
+
+                var nupkgPath = Path.Combine(workingDirectory, $"{projectName}.1.0.0.symbols.nupkg");
+                Assert.True(File.Exists(nupkgPath), "The output .nupkg is not in the expected place");
+                using (var nupkgReader = new PackageArchiveReader(nupkgPath))
+                {
+                    var files = nupkgReader.GetFiles();
+                    Assert.Contains(@"runtimes/win/lib/netstandard1.4/abc.pdb", files);
+                    Assert.Contains(@"runtimes/win/lib/net46/abc.pdb", files);
+                    Assert.DoesNotContain(@"lib/netstandard1.4/abc.pdb", files);
```

Comment:
Is the idea that net46 should have a lib folder?
Should we validate that?

Reply:
The idea here is that we can have custom build\symbol output without the actual build output. eg netstandard1.4 build output is not present but the custom symbol output (runtime file) is.

Reply:
My comment was more about line 2797:

<!--StartFragment-->                    ProjectFileUtils.AddProperty(xml, "IncludeBuildOutput", "false", $"'$(TargetFramework)'=='netstandard1.4'");<br class="Apple-interchange-newline"><!--EndFragment-->


The conditional suggests to me that we expect something different for the different frameworks.

Reply:
it seemed ```git commit ``` missed some of the files. Basically the build output files will be present for net46 and absent for netstandard1.4